packages/jest-types/src/Transform.ts

```diff
@@ -9,6 +9,5 @@
 export type TransformResult = {
   code: string;
   originalCode: string;
-  mapCoverage: boolean;
```

Comment:
can you make it optional instead of deleting it, and add a todo comment that we should delete it in Jest 26? It's a breaking change for people doing 

```ts
const result: TransformResult = {
  code: '',
  originalCode: '',
  mapCoverage: false.
}
```

as TS doesn't allow unknown fields in an object literal.

Reply:
Put this back as optional, with a TODO to remove.